syntax = "proto3";

message DeviceInfoRequest {

}
